mytimer/functions.py

```diff
@@ -239,20 +256,26 @@ def countup_timer(
     :type tone: str
     :param alarm_repeat: alarm repeat
     :type alarm_repeat: int
+    :param v_shift: vertical shift
+    :type v_shift: int
+    :param h_shift: horizontal shift
+    :type h_shift: int
     :return: None
     """
     timer_second = 0
     timer_minute = 0
     timer_hour = 0
+    prepend = ' ' * h_shift
     while True:
         start = time.perf_counter()
         clear_screen()
-        print('\n' * 5)
+        print('\n' * v_shift, end='')
         tprint(
-            '\t\t\t\t  %d : %d : %d ' %
-            (timer_hour,
-             timer_minute,
-             timer_second),
+            TIME_PRINT_TEMPLATE.format(
```

Comment:
Alternative :

```python
print(" " * h_shift, end='')
tprint(TIME_PRINT_TEMPLATE.format(timer_hour,timer_minute,timer_second), font=face, sep="\n" + " " * h_shift)
```

⚠️ We need to update the minimum version of `art` to `5.3`.